python/cudf/reshape/general.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+import numpy as np
+import pandas as pd
+
+from cudf.dataframe import Series
+from cudf.dataframe import Buffer
+from cudf.dataframe import DataFrame
+from cudf.utils import cudautils
+from cudf.dataframe.categorical import CategoricalColumn
+
+
+def melt(frame, id_vars=None, value_vars=None, var_name='variable',
+         value_name='value'):
+    """Unpivots a DataFrame from wide format to long format,
```

Comment:
I find this comment unclear. What does it mean to "unpivot"? Does one have to pivot first in order to unpivot? How does one create a pivoted data frame? What is wide format and what is long format?

I realize that this comment is lifted directly from pandas, but I don't find the pandas comment clear either.

Reply:
I really don't know how to describe what's being done here (or even why). I just mimicked pandas' functionality and thought that copying the docstring is appropriate. If a user is looking for melt, they probably know it from pandas and so if the docstring is same, it's a confirmation that it's the same thing.